<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookForm</class>
 <widget class="QWidget" name="AddBookForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Book</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>21</y>
     <width>561</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iSBNLabel">
      <property name="text">
       <string>ISBN</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="iSBNLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="titleLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="authorLabel">
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="authorLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="editionLabel">
      <property name="text">
       <string>Edition</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editionLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="publisherLabel">
      <property name="text">
       <string>Publisher</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="publisherLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="formatLabel">
      <property name="text">
       <string>Format</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="formatLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPlainTextEdit" name="DescriptionTextEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="publishingDateLabel">
      <property name="text">
       <string>Published</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDateEdit" name="publishingDateDateEdit">
      <property name="displayFormat">
       <string>dd/MM/yyyy</string>
      </property>
      <property name="date">
       <date>
        <year>1753</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="confirmation">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>164</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
